src/Filament/Resources/AuthorResource.php

```diff
@@ -31,12 +33,12 @@ public static function form(Form $form): Form
                 Fieldset::make()
                 ->schema([
                     TextInput::make('first_name')
-                        ->label(fn(Get $get) => $get('is_individual') ? 'First Name' : 'Organization Name')
+                        ->label(fn (Get $get) => $get('is_individual') ? 'First Name' : 'Organization Name')
```

Comment:
There is some linting happening that is distracting. Can we use prettier or editorconfig to document the desired standards?

Reply:
Yep.  I've addressed the unnecessary variable issue, but I'm creating a separate branch to deal with the linting.